ci-operator/step-registry/workers-rhel/pre-hook/sshtunnel/workers-rhel-pre-hook-sshtunnel-commands.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+
+export BASTION_SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+echo "RHEL VERSION: '${PLATFORM_VERSION}'"
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+declare bastion_private_address
+declare bastion_ssh_user
+if [ -f "${SHARED_DIR}"/bastion_private_address ]; then
+    bastion_private_address=$(< "${SHARED_DIR}"/bastion_private_address)
+else
+    echo "File bastion_private_address does not exist in SHARED_DIR, no need to deploy ssh tunnel service!"
+    exit 0
+fi
+if [ -f "${SHARED_DIR}"/bastion_ssh_user ]; then
+    bastion_ssh_user=$(< "${SHARED_DIR}"/bastion_ssh_user)
+else
+    echo "File bastion_ssh_user does not exist in SHARED_DIR, unable to deploy ssh tunnel service!"
+    exit 1
```

Comment:
Here the condition for `"${SHARED_DIR}"/bastion_ssh_user` and `"${SHARED_DIR}"/bastion_private_address` are different, per my understanding, they should be show up together, or either of them not show up, so I think the condition should be same.

Reply:
If my understanding is right, we can remove the line 118 to drop the `when` condition.

Reply:
correct, duplicated checks here, removed when condition and related block in ansible.